modules/fts/pages/fts-index-analyzers.adoc

```diff
@@ -12,20 +12,23 @@ An analyzer consists of modules, each of which performs a particular, sequenced
 _Analyzers_ pre-process input-text submitted for Full Text Search; typically, by removing characters that might prohibit certain match-options.
 
 The analysis is performed on document-contents when indexes are created and is also performed on the input-text submitted for a search.
-The benefit of analysis is often referred to as _language awareness_.
+This enables _language awareness_ for FTS..
 
-For example, if the input-text for a search is `enjoyed staying here`, and the document-content contains the phrase `many enjoyable activities`, the dictionary-based words do not permit a match.
-However, by using an analyzer that (by means of its inner _Token Filter_ component) _stems_ words, the input-text yields the tokens `enjoy`, `stay`, and `here`; while the document-content yields the tokens `enjoy` and `activ`.
-By means of the common token `enjoy`, this permits a match between `enjoyed` and `enjoyable`.
+For example, if the input-text for a search is `liked staying here`, and the document-content contains the phrase `many things to like`, the dictionary-based words do not permit a match.
+However, by using an analyzer (_en_ for this example) that _tokenizes_ and _stems_ words by means of its inner _Tokenizer_ and _Token Filter_ components, the input-text yields the tokens `like` and `stai`; while the document-content yields the tokens `mani`, `thing` and `like`.
+By means of the common token `like`, this permits a match between `like` and `liked`.
 
 Since different analyzers pre-process text in different ways, effective Full Text Search depends on the right choice of the analyzer, for the type of matches that are desired.
 
 Couchbase Full Text Search provides a number of pre-constructed analyzers that can be used with Full Text Indexes.
-Additionally, analyzers can be custom-created by means of the Couchbase Web Console.
+Additionally, analyzers can be custom-created on the Couchbase Web Console or using the http://bleveanalysis.couchbase.com/analysis[Text Analysis Wizard].
```

Comment:
http://bleveanalysis.couchbase.com/analysis => how easy it is integrate the exported json content of the custom analyser with that of an existing/new index definition?

IMO, it's very hard and error-prune.  So, not sure of mentioning it from a creation perspective.

Rather - if the bleve analysis tool had an import option to pass in a analysis JSONsection from a custom created analyzer of an index definition, then the user would be able to test it easily.?  (feature for bleve analysis tool)

Reply:
With the analysis wizard, the user will be able to export a custom analyzer definition. But yes pasting that JSON snippet into the index definition can be error prone.